external-crates/move/crates/move-compiler/src/expansion/ast.rs

```diff
@@ -1734,6 +1735,10 @@ impl AstDebug for ExpDotted_ {
                 w.comma(&rhs.value, |w, e| e.ast_debug(w));
                 w.write("]");
             }
+            D::DotUnresolved(_, e) => {
+                e.ast_debug(w);
+                w.write("")
```

Comment:
Nit: have this print something more clear

Reply:
It was supposed to be (fixed now):
```
                e.ast_debug(w);
                w.write("")
```